### ISSUE TITLE
SYSTEM_QUAZIP doesn't support Qt5

### ISSUE THREAD

**Issue by sebastic** [openstreetmap/merkaartor#66]
When building Merkaartor with Qt5 and enabling SYSTEM_QUAZIP (e.g. to use libquazip-qt5-dev on Debian/Ubuntu), linking fails because `src/src.pro` uses `-lquazip` unconditionally:
```
 14 isEmpty(SYSTEM_QUAZIP) {
 15   DEFINES += QUAZIP_STATIC
 16   include(../3rdparty/quazip-0.7/quazip.pri)
 17 } else {
 18   LIBS += -lquazip
 19 }
```
For Qt5 `-lquazip-qt5` should be used instead.

**Comment by Krakonos** [comment]
I haven't found in the quazip docs or sources where -qt5 is mentioned as a proper suffix to the library, and it will break other distros, as Gentoo, which has no such suffix. It looks like debian has done this to enable coexistence for qt4 and qt5 versions. Do you know of any mechanism that can be used to detect this?

**Comment by sebastic** [comment]
quazip has support to co-install the qt4 & qt5 libraries since [SVN r254](http://sourceforge.net/p/quazip/code/254/), it just appends "5" to the library name for Qt5.

The `-qt5` suffix is a recommendation from the Debian Qt maintainers to support both Qt4 and Qt5.

I'm not aware of any mechanism to detect this. Supporting a variable like SYSTEM_QUAZIP_LIBRARY to allow specifying the distribution specific library name may be a good option.

**Comment by Krakonos** [comment]
As there are no complaints, I consider this fixed in ba74296.
